Content.Shared/Construction/Steps/ConstructionGraphStepTypeSerializer.cs

```diff
@@ -31,6 +31,11 @@ public sealed class ConstructionGraphStepTypeSerializer : ITypeReader<Constructi
                 return typeof(TagConstructionGraphStep);
             }
 
+            if (node.Has("prototype"))
+            {
+                return typeof(PrototypeConstructionGraphStep);
+            }
```

Comment:
As per the graph step this really shouldn't exist, prototypes should just be for spawning and not be used as some authority on what an entity is.